### PR TITLE
Fixes mm-1912 move get file info into its own web service call

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: api/file.go [hunk 1]
```diff
@@ -281,11 +347,7 @@ func getFile(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	w.Header().Set("Cache-Control", "max-age=2592000, public")
-	w.Header().Set("Content-Length", strconv.Itoa(len(f)))
-
-	if r.Method != "HEAD" {
-		w.Write(f)
-	}
+	w.Write(f)
```
Comment (crspeller) [inline api/file.go]: What happened to the Content-Length header?

Comment (coreyhulen) [inline api/file.go]: content-length and content-type are set for us in w.Write()

Comment (crspeller) [inline api/file.go]: Cool.

**Comment by crspeller** [comment]
Looks good to me.

**Comment by nickago** [comment]
+1, ready to merge

**Pull request merged**